kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/dsl/operations/AbstractOperationDSL.kt

```diff
@@ -18,6 +18,11 @@ abstract class AbstractOperationDSL(
     internal var functionWrapper: FunctionWrapper<*>? = null
 
     private fun resolver(function: FunctionWrapper<*>): ResolverDSL {
+
+        require(function.hasReturnType()) {
+            "Resolver for $name has no return values"
```

Comment:
This is an issue I've hit a few times personally :smile: 

Could you change this message to `"Resolver for '$name' has no return value"`